docs/fast_data/configuration/config_maps/aggregation.md

```diff
@@ -727,41 +727,42 @@ The page will now show the list of fields, as defined in the Single View Data Mo
 
 Clicking on a field will open a drawer to the right side of the panel that allows you to map this field, by allowing the selection of an existing dependency and a field.
 
-![Detail of a field not mapped](../../img/no_code_aggregation/field_not_mapped.png)
+![Detail of a field not mapped](../img/no_code_aggregation/field_not_mapped.png)
 
 The _Dependency_ selection field will include a list of existing dependencies (on top) and all the other dependencies that can be connected (based on the conditions included inside the ER Schema). 
 
 Selecting a dependency will update the state of the _Set Path_ button and its label: if the dependency already exists, the label will inform the user that a "Path is set", indicating that the Aggregation already includes all the necessary dependencies (from the base projection - thus that dependency linked to the Projection Changes document via the `_identifier` condition) and it's already possible to select the field to complete the mapping: the _Field_  select field will now be available and the list will contain all the fields that can be used for the mapping.
 
 The selection will automatically create the mapping. This can be noted also by the fact that all the warnings next to the name of the field will disappear.
 
-![Detail of a dependency field](../../img/no_code_aggregation/dependency_field_detail.png)
+![Detail of a dependency field](../img/no_code_aggregation/dependency_field_detail.png)
 
 ### Automatic generation of dependencies via Path
 
 In case the value selected in the _Dependency_ select field is a dependency that does not exist yet, the "Path not set" text will still be there, and the `Set Path` button can be clicked to open a modal that will introduce the user to a new way to create dependencies.
 
-![Selection of the path](../../img/no_code_aggregation/path_selection.png)
+![Selection of the path](../img/no_code_aggregation/path_selection.png)
 
 The modal will include all the different paths to create all the needed dependencies to proceed with the mapping. Each path will represent a step composed of different projections connected via ER Schema `one-on-one` conditions. 
 
 In the example above, there are more ways to reach from _pr_registry_ to _pr_dishes_, and it will be possible to select the path needed for the mapping. The modal also includes a canvas to the left side, similar to the ER Schema canvas, that shows the currently selected modal to help the user to visualize the path and make the correct choice.
 
 Click on `Next` will move to the next step of the modal where the user will be prompted to select the condition for each step of the path.
 
-![Condition choices in selected path](../../img/no_code_aggregation/condition_selection_in_path.png)
+<!-- TODO: Update image (screenshot not present yet) -->
```

Comment:
We should use the image with the tooltip containing the condition, when released.

Reply:
Done